.github/workflows/publish.yml

```
@@ -1,53 +1,53 @@
+    strategy:
+      matrix:
+        release:
+          - type: Debug
+            suffix: '+debug'
+          - type: Release
+            suffix: ''
+
+    name: Build (${{ matrix.release.type }})
+
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)